packages/components/src/utils/button-handling.ts

```diff
@@ -1,19 +1,25 @@
 import type { ButtonType } from '../types';
-import { getClosestHTMLElement } from './dom';
+import { getClosestHTMLElement, setAttributes } from './dom';
 
 export const improveButtonHandlingForCustomElement = (
   element: HTMLElement,
   getType: () => ButtonType,
-  getDisabled: () => boolean
+  getDisabled: () => boolean,
+  getName?: () => string | undefined,
+  getValue?: () => string | undefined
```

Comment:
Could probably just capture these 4 with the rest operator like `...args` and then spread them on `handleButtonEvent` since the order is identical.

Reply:
Doesn't really make it easier because of typing